app/main.py

```diff
@@ -1,17 +1,9 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    """
-    Implement count_occurrences function:
```

Comment:
why did you delete docstring? you should write code after `# write your code here`